src/server.js

```diff
@@ -31,21 +32,20 @@ server.use('/public', express.static('public'));
 server.use(express.urlencoded({ extended: false }))
 server.use(flash())
 server.use(session({
-    //secret: process.env.SESSION_SECRET,
-    secret: "sami1234",
+    secret: process.env.SESSION_SECRET,
+    secret: process.env.SESSION_SECRET,
```

Comment:
is there supposed to be two secret vars?

Reply:
No, I had to cherry-pick, I left it there by mistake